AsyncDisplayKit/ASTextNode.h

```diff
@@ -26,6 +26,26 @@ typedef NS_ENUM(NSUInteger, ASTextNodeHighlightStyle) {
 };
 
 /**
+ * The vertical alignment of text within a node.
+ */
+typedef NS_ENUM(NSUInteger, ASTextNodeVerticalTextAlignment) {
+  /**
+   * Aligns the content vertically at the top in the node (the default).
+   */
+  ASTextNodeVerticalTextAlignmentTop,
```

Comment:
@nivanchikov my most important feedback for this diff is to find a way for this feature to be a bit less prominent, and not push down the rest of the functionality in this critical header as much.

Can you remove the block comments for each of the enum values, so there is one per line, and then on the same line as the first one do:

...Top,   // Top is the default value